dietpi/dietpi-software

```diff
@@ -5198,32 +5198,8 @@ _EOF_
 
 			Banner_Installing
 
-			INSTALL_URL_ADDRESS='https://dietpi.com/downloads/binaries/all/netdata_1.9.0_'
-
-			#armv6
-			if (( $G_HW_ARCH == 1 )); then
-
-				INSTALL_URL_ADDRESS+='armv6.deb'
-
-			#armv7+
-			elif (( $G_HW_ARCH == 2 )); then
-
-				INSTALL_URL_ADDRESS+='armv7.deb'
-
-			#ARMv8
-			elif (( $G_HW_ARCH == 3 )); then
-
-				INSTALL_URL_ADDRESS+='armv8.deb'
-
-			#amd64
-			elif (( $G_HW_ARCH == 10 )); then
-
-				INSTALL_URL_ADDRESS+='amd64.deb'
-
-			fi
-
 			DEPS_LIST='zlib1g-dev'
-			Download_Install "$INSTALL_URL_ADDRESS"
+			Download_Install "https://dietpi.com/downloads/binaries/all/netdata_1.11.1-1_$G_HW_ARCH_DESCRIPTION"
```

Comment:
@Fourdee 
What do you think about it to simplify arch based download links? I already renamed the files on server accordingly.

Reply:
@MichaIng 

Yep, nice one 👍

Reply:
At least when updating our binaries, we can apply this as well to other software titles.